newIDE/app/src/AssetStore/ExampleStore/index.js

```diff
@@ -55,7 +55,7 @@ export const prepareExampleShortHeaders = (
 
 const getItemUniqueId = (
   item: ExampleShortHeader | PrivateGameTemplateListingData
-) => item.name;
+) => (item.productType ? `template-${item.name}` : `example-${item.name}`);
```

Comment:
if we want to be 100% sure we have unique values, you could return `template-${item.tag}` and `example-${item.id}` ?